index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Eid Hachem</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet"> 
+</head>
+<body>
+    <header>
+        <nav class="head">
+            <p class="name">Eid Hachem</p>
+           <div class="menu">
+               <div class="menu-icon icon1"></div>
+               <div class="menu-icon icon2"></div>
+               <div class="menu-icon icon3"></div>
+           </div>
```

Comment:
- [ ] Kindly wrap the hamburger in the `nav tag`.

Reply:
Hi @sja-thedude 
It is already in a nav tag with the logo, should I put it inside another nav tag?